### PR TITLE
20479: Admin center: input json validation added.

### LINKED ISSUE #20479: Admin center - Javabatch: JSON input validation.
Add JSON input validation to APIs that are used in JavaBatch.

### PR DESCRIPTION
**Pull request by sarinjacobsunny**
[20479](https://github.com/OpenLiberty/open-liberty/issues/20479): Admin center - Javabatch: Input validation is required.

### CODE REVIEW

File: dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/rest/v1/CatalogAPI.java [hunk 1]
```diff
@@ -251,7 +251,7 @@ public Object deleteGrandchild(RESTRequest request, RESTResponse response, Strin
         if (!isAuthorizedDefault(request, response)) {
             throw new UserNotAuthorizedException();
         }
-
+
```
Comment (k8vance88) [inline dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/rest/v1/CatalogAPI.java]: This change looks unnecessary. Can we remove this?

File: dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java [hunk 2]
```diff
@@ -10,6 +10,32 @@
  *******************************************************************************/
 package com.ibm.ws.ui.internal.v1.utils;
 
+import com.google.gson.JsonParser;
+import com.google.gson.JsonSyntaxException;
+
+// import com.google.gson.Gson;
```
Comment (k8vance88) [inline dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java]: remove the commented out imports

File: dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java [hunk 3]
```diff
@@ -137,4 +163,37 @@ public synchronized static String getMD5String(String str) {
...
+            JsonParser parser = new JsonParser();
+            parser.parse(inputString);
+            isValid = true;
+        } catch(JsonSyntaxException e){
+            isValid = false;
+        } catch (Exception e) {
+            isValid = false;
+        }
+
+        // TODO: Delete sysout.
```
Comment (k8vance88) [inline dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java]: delete this

**Review by k8vance88 — CHANGES_REQUESTED** [review]
All files with copywrite statements need to have the latest date appended to the end. For example, if it says 2015 then you should add 2015, 2022.  If there are 2 dates, replace the last one with 2022.

**Comment by k8vance88** [comment]
Sarin, you should create a git issue and link it to for this PR.

File: dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/rest/v1/CatalogAPI.java (hunk 1 above)
Comment (sarinjacobsunny) [reply dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/rest/v1/CatalogAPI.java]: Removed extra space.

File: dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java (hunk 2 above)
Comment (sarinjacobsunny) [reply dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java]: Removed commented imports.

File: dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java (hunk 3 above)
Comment (sarinjacobsunny) [reply dev/com.ibm.ws.ui/src/com/ibm/ws/ui/internal/v1/utils/Utils.java]: Removed sysout

**Comment by sarinjacobsunny** [comment]
> All files with copywrite statements need to have the latest date appended to the end. For example, if it says 2015 then you should add 2015, 2022. If there are 2 dates, replace the last one with 2022.

Copyright texts updated.

**Comment by sarinjacobsunny** [comment]
> Sarin, you should create a git issue and link it to for this PR.

Open liberty ticket created.
https://github.com/OpenLiberty/open-liberty/issues/20479

**Comment by k8vance88** [comment]
#build

**Comment by k8vance88** [comment]
#run-libby-bot

**Review by k8vance88 — APPROVED** [review]

**Comment by sarinjacobsunny** [comment]
#build

**Comment by sarinjacobsunny** [comment]
#run-libby-bot

**Comment by sarinjacobsunny** [comment]
#build

**Comment by sarinjacobsunny** [comment]
#build

**Comment by sarinjacobsunny** [comment]
#build

**Comment by sarinjacobsunny** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_kIbTIPVwEey3m9ea3wJj4A

**Review by k8vance88 — APPROVED** [review]
LGTM

**Comment by sarinjacobsunny** [comment]
#run-libby-bot

**Comment by sarinjacobsunny** [comment]
#run-libby-bot

**Pull request merged**